official/utils/misc/distribution_utils.py

```diff
@@ -56,6 +58,10 @@ def get_distribution_strategy(distribution_strategy="default",
                        "cannot be set to 'off'.".format(num_gpus))
     return None
 
+  if distribution_strategy == "collective" or num_workers > 1:
+    return tf.contrib.distribute.CollectiveAllReduceStrategy(
```

Comment:
If Yuefeng submits his change to export the `CollectiveAllReduceStrategy` (and ParameterServerStrategy) in core TF soon, then perhaps this can be changed to use that and be 2.0 compatible?

Reply:
Sure, I will update after Yuefeng's change.